### ISSUE TITLE
PDF export is broken : slides overlap

### ISSUE THREAD

**Issue by Lucas-C** [hakimel/reveal.js#1252]
_C.f._ http://lab.hakim.se/reveal-js/?print-pdf
Visible in Chrome 43 & Firefox 38.
Sorry to not give more details, but I think the result after the link jump is self-explanatory.
I have no idea what is causing this, maybe css/print/pdf.css is broken ?

**Comment by hadim** [comment]
Same issue here !

**Comment by achauve** [comment]
+1

**Comment by waywaaard** [comment]
Cannot reproduce this? In Chrome 43 the slides do not overlap in print preview? Do you really use print preview to see the final layout of the slides for printing? The pure HTML presentation of the print layout does not show the real output.

**Comment by saimn** [comment]
I have this issue when I use `data-background` with an empty slide (just to show a fullscreen image). Then the background image is shown under the next slide when printing.

**Comment by Lucas-C** [comment]
Ok so here is what it looks like under chrome 43 for me:

![reveal-js_chrome43_initial_tmp_appearance](https://cloud.githubusercontent.com/assets/925560/8101828/171e0d1a-101b-11e5-9bea-6d69909e4457.png)

The screen above only appears during a second, and then I get the following (still broken) final screen :

![reveal-js_chrome43_final_appearance](https://cloud.githubusercontent.com/assets/925560/8101843/48801312-101b-11e5-862c-9495ede2beef.png)

**Comment by waywaaard** [comment]
But this is not the print preview! You need to adjust the page format and the borders for the final result.

![image](https://cloud.githubusercontent.com/assets/1634468/8101935/48b493ac-101c-11e5-95d4-02f0eac880db.png)

**Comment by hakimel** [comment]
@waywaaard is correct. The overlapping content doesn't matter for the final PDF output – it will only look like that before you open the print dialog.

**Comment by saimn** [comment]
With
```
<section data-background="https://s3.amazonaws.com/hakim-static/reveal-js/image-placeholder.png">
</section>
<section data-background="https://s3.amazonaws.com/hakim-static/reveal-js/image-placeholder.png" data-background-repeat="repeat" data-background-size="100px">
....
```
the background overlaps with the next slide content:
![2015-06-11-103436_1680x1050_scrot](https://cloud.githubusercontent.com/assets/311639/8103052/aff0b402-1025-11e5-9198-9dfa175b3c5c.png)
This can be avoided by adding some fake content in the empty section (`<div>&nbsp;</div>`), but it can be surprising when discovering reveal.js

**Comment by hakimel** [comment]
@saimn That looks like a bug, though unrelated to this issue. Would you mind opening a new issue for it?

**Comment by Lucas-C** [comment]
My mistake about the original report, sorry.
And thanks for helping me !

**Comment by tomwestwood** [comment]
Afternoon all :)

I'll try and keep it as short as possible as this might be already answered above as it's very similar symptoms (more severe in my case; see below screen shot). I've tried the above and it hasn't provided the desired solution; print preview still displays overlapping elements. I'm using a dummy presentation as a concept to get the fix before I apply to ours.

How it should look:

![how_it_should_look](https://cloud.githubusercontent.com/assets/12938753/8211918/e49528c4-1510-11e5-9f3a-0c7024c1f7c3.jpg)

How it does look when I apply the '?print-pdf' extension onto the presentation URL:

![how_it_does_look](https://cloud.githubusercontent.com/assets/12938753/8211928/ed2b2cb8-1510-11e5-80ca-45145849bf3c.jpg)

And in print preview:

![how_it_does_look_pp](https://cloud.githubusercontent.com/assets/12938753/8211982/2856e1e2-1511-11e5-80da-3fe62027316f.jpg)

I'm using Chrome (v43) and I'm pretty sure I'm using the latest reveal source files, so is there anything you guys can think of that might be causing the above? Any assistance would be greatly appreciated :)

Regards

Tom

**Comment by AbduKT** [comment]
+1

**Comment by johanfrisk** [comment]
Hi, any hints on why I get "slide-1" inserted at the end of the search string and thus only can pdf one slide at a time? Running Chrome (v47).

**Comment by Godrebh** [comment]
I had the same problem and could not solve it. Eventually I switched to [decktape](https://github.com/astefanutti/decktape), which works like a charm.

**Comment by llaville** [comment]
Thanks @Godrebh . Interresting feedback about decktape !

**Comment by ski11io** [comment]
Same Problem here. +1

**Comment by hakimel** [comment]
From the look of of the screenshots in this thread, the [css/print/pdf.css](https://github.com/hakimel/reveal.js/blob/master/css/print/pdf.css) isn't loading. Can you try manually including it in your HTML file and printing again?

That stylesheet is normally loaded using a small script in the head of the default index.html file: https://github.com/hakimel/reveal.js/blob/master/index.html#L16-L22. However that may stop working if the pdf.css or index.html files are moved to a different relative path or the script is removed from the index.html file.

**Comment by ski11io** [comment]
Solved! Thx

**Comment by ilanvandinski** [comment]
Hi, 

It seems that I have a similar problem. I have added the pdf.css file and I've tried to print my presentation by adding the print-pdf parameter, and it's working for the slides with divs whose width is of 100%, but if I have several divs with other width it's not working. 

An example with a width of 100%:

![image](https://cloud.githubusercontent.com/assets/20731093/18662558/25f76172-7f1b-11e6-9a4d-0c0b43ff22b3.png)

And another one with various width:

![image](https://cloud.githubusercontent.com/assets/20731093/18662596/553c91e6-7f1b-11e6-8775-1bb98a4d839d.png)

Did anyone have the same problem?
